specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -0,0 +1,2200 @@
+      "enum": [
+        "PUBLISHED"
+      ],
+      "x-ms-enum": {
+        "name": "PublishStatus",
+        "modelAsString": false
+      }
+    },
+    "ArtifactType": {
+      "description": "Artifact type.",
+      "type": "string",
+      "enum": [
+        "DATABASE",
+        "TABLE",
+        "SCHEMA",
+        "VIEW",
+        "FUNCTION",
+        "PARTITIONINFO",
+        "RELATIONSHIP"
+      ],
+      "x-ms-enum": {
+        "name": "SASEntityType",
+        "modelAsString": false
+      }
+    },
+    "TableType": {
+      "description": "Entity type.",
+      "type": "string",
+      "enum": [
+        "MANAGED",
+        "EXTERNAL"
+      ],
+      "x-ms-enum": {
+        "name": "TableType",
+        "modelAsString": false
+      }
+    },
+    "SortOrder": {
+      "description": "Sorting order.",
+      "type": "string",
+      "enum": [
+        "DESC",
+        "ASC"
+      ],
+      "x-ms-enum": {
+        "name": "SortOrder",
+        "modelAsString": false
+      }
+    },
+    "RelationshipType": {
+      "description": "Relation Type",
+      "type": "string",
+      "enum": [
+        "ONETOONE",
+        "ONETOMANY",
+        "MANYTOONE",
+        "MANYTOMANY"
+      ],
+      "x-ms-enum": {
+        "name": "RelationshipType",
+        "modelAsString": false
+      }
+    },
+    "ValidationType": {
+      "description": "Validation Type",
+      "type": "string",
+      "enum": [
+        "IDWValidation"
+      ],
+      "x-ms-enum": {
+        "name": "ValidationType",
+        "modelAsString": false
+      }
+    },
+    "ValidationStatus": {
+      "description": "Validation Status",
+      "type": "string",
+      "enum": [
+        "VALID",
+        "INVALID"
+      ],
+      "x-ms-enum": {
+        "name": "ValidationStatus",
+        "modelAsString": false
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "description": "The object that defines the structure of an Azure Synapse error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "x-ms-client-flatten": true,
+          "description": "Error data",
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "description": "The object that defines the structure of an Azure Synapse error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "Property name/path in request associated with error.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Array with additional error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudError"
```

Comment:
This reference likely be wrong. It should ref to `CloudErrorBody`. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#Handling-Errors

Reply:
This model is exactly the same as what Synapse is already using.
Refer - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/synapse/common/v1/types.json

Reply:
Then why do not refer to that object in common/v1/types.json?